x-pack/test/functional/apps/grok_debugger/home_page.ts

```diff
@@ -0,0 +1,111 @@
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const PageObjects = getPageObjects(['common', 'grokDebugger']);
+  const browser = getService('browser');
+  const security = getService('security');
+  const testSubjects = getService('testSubjects');
+  const retry = getService('retry');
+
+  describe('Grok Debugger', function () {
+    before(async () => {
+      // await esArchiver.load('x-pack/test/functional/es_archives/empty_kibana');
+      // Increase window height to ensure "Simulate" button is shown above the
+      // fold. Otherwise it can't be clicked by the browser driver.
+      await browser.setWindowSize(1600, 1000);
+      await security.testUser.setRoles(['global_devtools_read', 'ingest_pipelines_user']);
+      await PageObjects.common.navigateToApp('grokDebugger');
+      await retry.waitFor('Grok Debugger Header to be visible', async () => {
+        return testSubjects.isDisplayed('grokDebuggerContainer');
+      });
+    });
+
+    it('Loads the app', async () => {
+      await retry.waitForWithTimeout('Grok Debugger to be visible', 15000, async () => {
+        return await (await PageObjects.grokDebugger.simulateButton()).isDisplayed();
+      });
+    });
+
+    it('Accept and parse input with built in grok pattern', async () => {
+      const eventInput = 'SegerCommaBob';
+      const patternInput = '%{USERNAME:u}';
+      const response = await PageObjects.grokDebugger.executeGrokSimulation(
+        eventInput,
+        patternInput,
+        null
+      );
+      expect(response).to.eql({ u: 'SegerCommaBob' });
+    });
+
+    it('Accept and parse input with custom in grok pattern', async () => {
+      await PageObjects.common.navigateToApp('grokDebugger');
+      const eventInput =
+        'Jan  1 06:25:43 mailserver14 postfix/cleanup[21403]: BEF25A72965: message-id=<20130101142543.5828399CCAF@mailserver14.example.com>';
+      const patternInput = '%{SYSLOGBASE} %{POSTFIX_QUEUEID:queue_id}: %{MSG:syslog_message}';
+      const customPatternInput = 'POSTFIX_QUEUEID [0-9A-F]{10,11}\nMSG message-id=<%{GREEDYDATA}>';
+      const testData = {
+        pid: '21403',
+        program: 'postfix/cleanup',
+        logsource: 'mailserver14',
+        syslog_message: 'message-id=<20130101142543.5828399CCAF@mailserver14.example.com>',
+        queue_id: 'BEF25A72965',
+        timestamp: 'Jan  1 06:25:43',
+      };
+
+      const response = await PageObjects.grokDebugger.executeGrokSimulation(
+        eventInput,
+        patternInput,
+        customPatternInput
+      );
+      expect(response).to.eql(testData);
+    });
+
+    // This test will need to be fixed.
```

Comment:
Can you explain what's wrong with this test and why we need to still skip it?

Reply:
The UI has changed since this test was last working. I will need to actually rewrite this test from the ground up. The current PR is just about migrating the existing tests to use current conventions to aid with flakiness. So fixing this other test should be fixed as part of a separate since that wasn't the purpose of this PR.

Reply:
Thanks for the explanation! As discussed 1:1, if you can open a separate issue to address this so we do lose track of it that'd be great. It'd also be helpful if you can reference the issue link in the code comment.